packages/messenger-widget/src/hooks/conversation/useConversation.tsx

```diff
@@ -1,18 +1,20 @@
 /* eslint-disable max-len */
-import { normalizeEnsName } from '@dm3-org/dm3-lib-profile';
 import { Conversation } from '@dm3-org/dm3-lib-storage/dist/new/types';
 import { useContext, useEffect, useMemo, useState } from 'react';
-import { fetchPendingConversations } from '../../adapters/messages';
 import { AuthContext } from '../../context/AuthContext';
 import { StorageContext } from '../../context/StorageContext';
-import { TLDContext } from '../../context/TLDContext';
-import { Config } from '../../interfaces/config';
 import { ContactPreview, getDefaultContract } from '../../interfaces/utils';
 import { useMainnetProvider } from '../mainnetprovider/useMainnetProvider';
 import { hydrateContract } from './hydrateContact';
+import { fetchPendingConversations } from '../../adapters/messages';
+import { normalizeEnsName } from '@dm3-org/dm3-lib-profile';
+import { Config } from '../../interfaces/config';
+import { TLDContext } from '../../context/TLDContext';
+import { useDm3Configuration } from '../configuration/useDM3Configuration';
 
 export const useConversation = (config: Config) => {
     const mainnetProvider = useMainnetProvider();
+    const { dm3Configuration } = useDm3Configuration();
```

Comment:
I still think the configuration will always be empty since the invocation of the hook loads the default state. i think the hook has to be exported similarly to MessageContext to make it globally accessible